.github/actions/verify-token/dist/dist/index.js

```diff
@@ -64,7 +64,7 @@ function run() {
             /* Test locally. Requires a GitHub token:
                 $ env INPUT_SLSA-WORKFLOW-RECIPIENT="delegator_generic_slsa3.yml" \
                 INPUT_SLSA-UNVERIFIED-TOKEN="$(cat testdata/slsa-token)" \
-                INPUT_TOKEN="$(gh auth token)" \
+                INPUT_TOKEN="$(echo $GH_TOKEN)" \
```

Comment:
any reason to switch this? i found it easiest to rely on the `gh` CLI

Reply:
let me undo it. For some reason it was no long working locally.. could be because I was not authenticated.